problems/isMultipleOfFive.js

```diff
@@ -13,7 +13,14 @@
 * returns: false (because the string "ten" is not a number)
 */
 
-function isMultipleOfFive() {
+function isMultipleOfFive(n) {
+    if (n % 5 === 0) {
+        return true;
+    } else {
+        if (NaN || !(n % 5 === 0)) {
+            return false
+        }
```

Comment:
we could of just returned false inside of the else statement. We can assume it is not a multiple of 5 if it failed the if statement

Reply:
copy